sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -689,6 +689,11 @@ case class DescribeColumnCommand(
       buffer += Row("distinct_count", cs.map(_.distinctCount.toString).getOrElse("NULL"))
       buffer += Row("avg_col_len", cs.map(_.avgLen.toString).getOrElse("NULL"))
       buffer += Row("max_col_len", cs.map(_.maxLen.toString).getOrElse("NULL"))
+      buffer ++= cs.flatMap(_.histogram.map { hist =>
```

Comment:
I'm pretty sure that for-comprehension would make the code read easier.

```scala
for {
  c <- cs
  hist <- c.histogram
  ...
} yield ...
```

Let me know if you need help with that.

Reply:
Thanks for your feedback. IMHO I am not sure that it would be easier to read. But if someone else agrees with you I can change it, thanks.

Reply:
yea I think for this case the for loop is conventional